### ISSUE TITLE
Movies don't seem to be recognised

### ISSUE THREAD

**Issue by ppfeufer** [MediaBrowser/Emby.AutoOrganize#41]
TV shows work just fine, but with movies, I do have problems getting it to work at all. No matter which movie I try, I always get this message:

![image](https://user-images.githubusercontent.com/2989985/155507959-e2f09f33-f9aa-42ef-86d1-d4c6f9aaae2b.png)

When moving the movie manually into the "Movies" directory, it will be recognised just fine after the library update is running through, but not via Auto Organize.

My configuration:

![image](https://user-images.githubusercontent.com/2989985/155508077-4de3e38b-da95-45c1-a33d-e38f7cf53b2b.png)

**Comment by LukePulverenti** [comment]
HI, can you please attach the emby server log from when you tried to do this? Thanks.

**Comment by ppfeufer** [comment]
Sure thing:

```
2022-02-25 18:03:33.539 Info Server: http/1.1 POST http://emby.ppfeufer.de/ScheduledTasks/Running/1ec5b785c4111045fc22055e467e2fff?X-Emby-Client=Emby Web&X-Emby-Device-Name=Chrome&X-Emby-Device-Id=9492ddfb-bc70-4db5-8ed3-050f2e8f618d&X-Emby-Client-Version=4.6.7.0. UserAgent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36
2022-02-25 18:03:33.568 Info Server: http/1.1 Response 204 to 192.168.178.23. Time: 29ms. http://emby.ppfeufer.de/ScheduledTasks/Running/1ec5b785c4111045fc22055e467e2fff?X-Emby-Client=Emby Web&X-Emby-Device-Name=Chrome&X-Emby-Device-Id=9492ddfb-bc70-4db5-8ed3-050f2e8f618d&X-Emby-Client-Version=4.6.7.0
2022-02-25 18:03:33.570 Info App: Sorting file /volume1/utgard/media/video/incoming/movies/AVP_Alien_vs_Predator.mkv
2022-02-25 18:03:33.571 Warn App: Unable to find movie in library matching name AVP_Alien_vs_Predator.mkv
2022-02-25 18:03:33.640 Info Server: http/1.1 GET http://emby.ppfeufer.de/ScheduledTasks?IsEnabled=true&X-Emby-Client=Emby Web&X-Emby-Device-Name=Chrome&X-Emby-Device-Id=9492ddfb-bc70-4db5-8ed3-050f2e8f618d&X-Emby-Client-Version=4.6.7.0.UserAgent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36
2022-02-25 18:03:33.642 Info Server: http/1.1 Response 200 to 192.168.178.23. Time: 2ms. http://emby.ppfeufer.de/ScheduledTasks?IsEnabled=true&X-Emby-Client=Emby Web&X-Emby-Device-Name=Chrome&X-Emby-Device-Id=9492ddfb-bc70-4db5-8ed3-050f2e8f618d&X-Emby-Client-Version=4.6.7.0
```

**Comment by pelaxa** [comment]
any movement on this?  I have the same issue and the reason I started using this plugin was just to get the movies organized into folders of the same name so I can support multiple res, etc.  So I have taken my Movies from the library to an import folder and trying to import them into a new Movie Library.  All the files are named in a way that Emby itself understands them but this plugin fails to import any of them.

**Comment by LukePulverenti** [comment]
> any movement on this? I have the same issue and the reason I started using this plugin was just to get the movies organized into folders of the same name so I can support multiple res, etc. So I have taken my Movies from the library to an import folder and trying to import them into a new Movie Library. All the files are named in a way that Emby itself understands them but this plugin fails to import any of them.

Are you saying you have auto organize watching an existing library folder? If so then that’s not supported. If not then please provide a specific example and attach the Emby server log. Thanks.

**Comment by pelaxa** [comment]
> > any movement on this? I have the same issue and the reason I started using this plugin was just to get the movies organized into folders of the same name so I can support multiple res, etc. So I have taken my Movies from the library to an import folder and trying to import them into a new Movie Library. All the files are named in a way that Emby itself understands them but this plugin fails to import any of them.
> 
> Are you saying you have auto organize watching an existing library folder? If so then that’s not supported. If not then please provide a specific example and attach the Emby server log. Thanks.

No.  I created a new library and moved my movies to a new location that is being watched by auto organize.  All the movies are named like "<movie name> (<year>).ext" and when I go to manually match the movie (after it has failed) it is parsing the title and year properly and then finds the movie but it all fails when auto scanned with the same error as in this ticket.

I was expecting the plugin to not only add existing movies of different resolution, but also be able to add any new movies (not in library) that I drop into the folder.

I have the exact same setup as above.

**Comment by LukePulverenti** [comment]
> > > any movement on this? I have the same issue and the reason I started using this plugin was just to get the movies organized into folders of the same name so I can support multiple res, etc. So I have taken my Movies from the library to an import folder and trying to import them into a new Movie Library. All the files are named in a way that Emby itself understands them but this plugin fails to import any of them.
> > 
> > 
> > Are you saying you have auto organize watching an existing library folder? If so then that’s not supported. If not then please provide a specific example and attach the Emby server log. Thanks.
> 
> No. I created a new library and moved my movies to a new location that is being watched by auto organize. All the movies are named like " ().ext" and when I go to manually match the movie (after it has failed) it is parsing the title and year properly and then finds the movie but it all fails when auto scanned with the same error as in this ticket.
> 
> I was expecting the plugin to not only add existing movies of different resolution, but also be able to add any new movies (not in library) that I drop into the folder.
> 
> I have the exact same setup as above.

server log example?
